README.md

```diff
@@ -54,7 +54,7 @@ the request log at `localhost:9191/__admin/requests/` and a JSON representation
 ### Starting feature tests node instance
 
 A separate node instance needs to be started for the feature tests. This will run on port 3008 and won't conflict
-with any of the api services, e.g. elite2-api or oauth. It will also not conflict with the Groovy integration tests.
+with any of the api services, e.g. prisonapi-api or oauth. It will also not conflict with the Groovy integration tests.
```

Comment:
probably don't need the -api here . I think we generally type in camel case as well, so `prisonApi`